Settings/CWApiSettings.cs

```diff
@@ -43,31 +48,34 @@ public class CWApiSettings
 
         /// <param name="domain">The base URL of your ConnectWise Manage installation. Example: https://cw.siteurl.com</param>
         /// <param name="companyName">The 'short' Company Name that you use when you login to ConnectWise Manage.</param>
+        /// <param name="clientId">The clientId generated from ConnectWise documentation: https://developer.connectwise.com/ClientID#What_is_a_clientId.3F </param>
         /// <param name="publicKey">The public API key - setup with Member Authentication.</param>
         /// <param name="privateKey">The private API key - setup with Member Authentication.</param>
         /// <param name="uriVersion">The version that appears in the Uri.</param>
         /// <param name="accept">The Accept Header value.</param>
-        public CWApiSettings(string domain, string companyName, IAuthSettings auth, string uriVersion = "3.0", string accept = "application/json")
+        public CWApiSettings(string domain, string companyName, string clientId, IAuthSettings auth, string uriVersion = "3.0", string accept = "application/json")
         {
-            init(domain, companyName, auth, uriVersion, accept);
+            init(domain, companyName, clientId, auth, uriVersion, accept);
         }
 
         /// <param name="domain">The base URL of your ConnectWise Manage installation. Example: https://cw.siteurl.com</param>
         /// <param name="companyName">The 'short' Company Name that you use when you login to ConnectWise Manage.</param>
+        /// /// <param name="clientId">The clientId generated from ConnectWise documentation: https://developer.connectwise.com/ClientID#What_is_a_clientId.3F </param>

```

Comment:
Sorry I'm a little anal about this but fix the extra forward slashes in this line and I will approve this PR and make sure it goes out at the same time as the issue you opened. Thank you!